server/api/commands.go

```diff
@@ -37,12 +35,13 @@ func LoadTest(w stdio.Writer, state State, waitForCancel func(), throttle string
 	frames, err := atoi(strcoll.Nth(3, cmd), err)
 	conc, err := atoi(strcoll.Nth(4, cmd), err)
 	qps, err := atoi(throttle, err)
-	reqBody := compose.TransactionRequest(events, spans, frames)
-	url := "/v1/transactions"
+
+	url := "/v2/intake"
```

Comment:
does this get `/events` appended later?

Reply:
no, don't think so, good catch 👍 
still have to test this